060_Distributed_Search/20_Scan_and_scroll.asciidoc

```diff
@@ -1,14 +1,11 @@
 [[scan-scroll]]
```

Comment:
I have no strong opinion here, it's probably the path of least resistance to leave the blockID as-is ... redirects are always temperamental anyway.